client/my-sites/email/email-management/home/email-list-active.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { CompactCard } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'calypso/components/section-header';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import EmailTypeIcon from 'calypso/my-sites/email/email-management/home/email-type-icon';
+import { getNumberOfMailboxesText } from 'calypso/my-sites/email/email-management/home/utils';
+
+class EmailListActive extends React.Component {
+	render() {
+		const { selectedSiteSlug, currentRoute, domains, translate } = this.props;
+
+		if ( domains.length < 1 ) {
+			return null;
+		}
+
+		const headerCard = <SectionHeader label={ translate( 'Active email plans' ) } />;
```

Comment:
Yes, although I would rather avoid the term _active_ as it may mean different things depending on the context. Another option could be `Domains with emails`.